pubspec.lock

```diff
@@ -306,6 +306,6 @@ packages:
       name: yaml
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.10"
+    version: "2.1.12"
 sdks:
-  dart: ">=1.17.0-dev.6.2 <2.0.0"
+  dart: ">=1.21.0-dev.2.0 <2.0.0"
```

Comment:
Hmm, good question; I didn't look closely enough and assumed this was the pubspec.yaml file.

It looks like what you really need is a slight bump to the markdown package? In which case, you could revert the changes to this file, run `pub get`, and it'll jump pull down the min. necessary to satisfy the changed deps.

You probably ran `pub upgrade`, which is more of a move-everything-to-the-latest operation.